assets/stylesheets/common/docker-manager.scss

```diff
@@ -0,0 +1,155 @@
+.dashboard-stats .upgrades-banner {
+  background: var(--tertiary-low);
+  flex: 1 1 100%;
+  margin-bottom: 2rem;
+  padding: 1rem;
+
+  &.upgrade-needed {
+    background: var(--danger-low);
+
+    a {
+      color: var(--danger);
+    }
+  }
+
+  a {
+    font-weight: bold;
+  }
+}
+
+.docker-manager {
+  h1 {
+    margin-top: 1rem;
+  }
+
+  .console-logs {
+    background-color: black;
+    color: #ddd;
+    font-family: monospace;
+    height: 400px;
+    margin-top: 20px;
+    overflow-x: scroll;
+    padding: 10px;
+    white-space: pre;
+  }
+
+  .check-circle {
+    color: var(--success);
+  }
+
+  table#repos {
+    margin-top: 10px;
+
+    td:first-child {
+      width: 3%;
+      padding-left: 8px;
+    }
+
+    th {
+      border-top: none;
+    }
+
+    td,
+    th {
+      padding: 8px 15px 8px 0px;
+    }
+
+    button {
+      padding-left: 15%;
+      padding-right: 15%;
+    }
+  }
+
+  .new-version {
+    h4 {
+      margin: 0 0 10px 0;
+      font-weight: bold;
+      font-size: 14px;
+    }
+
+    ul {
+      list-style: none;
+      margin: 0 0 10px 0;
+      padding: 0;
+    }
+
+    li.new-commits {
+      font-style: italic;
+    }
+  }
+
+  span.commit-hash {
+    color: #959595;
+  }
+
+  button.unlock {
```

Comment:
Nitpick, can certainly address later, but this could use `.btn-danger` instead now?